sql/core/src/test/resources/sql-tests/inputs/pgSQL/with.sql

```diff
@@ -0,0 +1,1216 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- WITH
+-- https://github.com/postgres/postgres/blob/REL_12_BETA1/src/test/regress/sql/with.sql
+
+--
+-- Tests for common table expressions (WITH query, ... SELECT ...)
+--
+
+-- Basic WITH
+-- [ORIGINAL SQL]
+--WITH q1(x,y) AS (SELECT 1,2)
+--SELECT * FROM q1, q1 AS q2;
+WITH q1(x,y) AS (SELECT 1,2)
+SELECT * FROM q1 CROSS JOIN q1 AS q2;
```

Comment:
According to the `case.sql`, it seems that we prefer to use `crossJoin.enabled` at the beginning and at the end.
```
set spark.sql.crossJoin.enabled=true;
...
set spark.sql.crossJoin.enabled=false;
```

Reply:
thanks @dongjoon-hyun, I've changed it.